code/modules/research/designs/autolathe_designs.dm

```diff
@@ -279,6 +279,7 @@
 	materials = list(/datum/material/iron = 1000)
 	build_path = /obj/item/tank/internals/generic
 	category = list("initial","Misc","Equipment")
+	departmental_flags = DEPARTMENTAL_FLAG_ENGINEERING | DEPARTMENTAL_FLAG_CARGO
```

Comment:
uh, was this change supposed to be a part of this PR?

Reply:
Is it a part of another merged PR because that's the only reason it should be?


AAAAAAAAAAAA

Reply:
Fixed